internal/pkg/config/testdata/src/notexample.com/exclusion/test.go

```diff
@@ -0,0 +1,19 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package allowlisting
```

Comment:
Nit: Package name does not match package path.

Reply:
Oh woops. Good catch! This is a remnant from an earlier angle I took. I was originally going in with "allowlist" in mind, but then I changed it to "exclude" because it was more coherent with what the code was actually doing. I missed this one when I did the renaming :upside_down_face: